blog-highlights/demystifying-appflowy-editors-codebase.md

```diff
@@ -355,43 +336,44 @@ Here is an example of a transaction used in the _remove\_word\_command.dart_ fil
 
 In the above code, you can see that we are calling a method called `deleteText` of the class Transaction. Similarly, there are other useful methods in this class, which change the content and state of the document.
 
-Explore the class by looking in:
+Explore the class by looking in: 
 
 ```
 lib\src\core\transform\transaction.dart
 ```
 
-### src/extensions
+To learn more about how Transaction manages State Refresh read this [article](https://blog-appflowy.ghost.io/how-we-built-a-highly-customizable-rich-text-editor-for-flutter/).
 
+### src\extensions
 This directory consists of helpful extensions on Attributes, Colors, TextSpan, Node, Position, and much more.
 
-### src/plugins
+### src\plugins
 
 This is a family of plugins that make data encoding and decoding possible with popular formats such as HTML, Markdown, and Quill Delta. We use a `Document` class to convert the contents of our Editor to these formats.
 
-Here are the contents:\
-\
-`src/plugins`\
-`├── html`\
-`├── markdown`\
-`├── quill_delta`\
-`└── plugins.dart`\\
+Here are the contents: \
+   \
+   `src\plugins` \
+   `├── html` \
+   `├── markdown` \
+   `├── quill_delta` \
+   `└── plugins.dart`
+  
 
 It is essential that we allow the contents of our Document to be converted to these formats because we want to allow our data to be synced across the web and other tools. These plugins are how we achieve this.
 
-That's concludes our disucssion on the folder organization of the AppFlowyEditor package.
+That concludes our disucssion on the folder organization of the AppFlowyEditor package.
```

Comment:
```suggestion
That concludes our discussion on the folder organization of the AppFlowyEditor package.
```

Reply:
some of these typos sound really funny, thanks for pointing them out. DISUCSSION LOL